src/Compute/Compute/help/New-AzVM.md

```diff
@@ -212,7 +210,7 @@ $bytes = [System.Text.Encoding]::Unicode.GetBytes($text);
 $userData = [Convert]::ToBase64String($bytes);
 
 # Create VM
-New-AzVM -ResourceGroupName $ResourceGroupName -Name $VMName -Credential $Credential -DomainNameLabel $domainNameLabel -UserData $userData -SecurityType $securityTypeStnd;
```

Comment:
Have you walked through all of these examples and run them to verify that they function with Standard removed?

Reply:
Validated 3 examples, can confirm validation for all examples tomorrow.

Reply:
@Sandido - Validated all examples.
Removed example 9 from the list as well to ensure TL-as-default is not perceived as invasive.